src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -118,6 +159,20 @@ bool RPCServiceImpl::ManageMobileCommand(
       SendMessageToMobile(response);
       return false;
     }
+    if (EncryptionFlagCheckResult::kError_EncryptionNeeded ==
+        IsEncryptionRequired(
+            *message,
+            app,
+            protocol_handler_->IsRPCServiceSecure(connection_key))) {
+      const auto response = MessageHelper::CreateNegativeResponse(
+          connection_key,
+          function_id,
+          correlation_id,
+          static_cast<int32_t>(mobile_apis::Result::ENCRYPTION_NEEDED));
+      ;
```

Comment:
Remove extra semicolon?

Reply:
fixed in 6a23e0c4b08b374cefd63c236415371a0b0e2a32